### PR TITLE
🚀 Chat: Initialized A New Empty App

### LINKED ISSUE #45: 🚀 Adding a Feature: Chat - Basic Setup
## Setup A New Basic *Chat*-App, Without Models And Forms

- [x] Initialize a new empty app designated for *chat*,
       by executing the command:
       ```
       pipenv run python manage.py startapp chat
       ```
- [x] Set the `app/settings.py` to enable templates
       for the *chat*-app.
- [x] An an option for the *chat*-app to be viewed at:
       http://localhost:8000/chat/ with an initial
       `.html` file.

### PR DESCRIPTION
**Pull request by taljacob2**
1. Initialized a new empty app designated for *chat*,
    by executing the command:
    ```
    pipenv run python manage.py startapp chat
    ```
1. Set the `app/settings.py` to enable templates
    for the *chat*-app.
1. The *chat*-app can be viewed now at:
    http://localhost:8000/chat/ with an initial
    `.html` file.
    > NOTE: This `.html` is very minimal.
    > It is only to validate that the route to
    > http://localhost:8000/chat/ works successfully,
    > for future work.
    > 
    > image

---

DEPENDS ON #34 
Close #45 

Signed-off-by: Tal Jacob <taljacob2@gmail.com>

### CODE REVIEW

**Review by paOmer — APPROVED** [review]
LGTM!

**Review by DeanBiton — APPROVED** [review]
LGTM

**Comment by Yarboa** [comment]
Note there are merge conflicts,
Please rebase and push

**Comment by taljacob2** [comment]
> Note there are merge conflicts, Please rebase and push

Resolved conflicts now. You may review.

File: chat/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# from django.contrib import admin
```
Comment (Yarboa) [inline chat/admin.py:1]: Is it needed?

**Review by Yarboa — COMMENTED** [review]
Please think of a test to verify what ever exist here is working.
Please refer last tests talk

File: chat/admin.py (hunk 1 above)
Comment (taljacob2) [reply chat/admin.py:1]: > Please think of a test to verify what ever exist here is working. Please refer last tests talk

Added a simple test in [4cf4d57](https://github.com/redhat-beyond/JobSeeker/pull/43/commits/4cf4d5762c6aa801dab4693f09e6a0472c377646).


> Is it needed?

That line is commented-out so *Flake8* could approve.
Do you prefer to remove this line entirely?

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**